app/src/python/user-scripts/trf2dcm.py

```diff
@@ -0,0 +1,32 @@
+import os
+from glob import glob
+
+import pydicom
+
+from pymedphys.deliverydata import delivery_data_to_dicom
+from pymedphys.trf import delivery_data_from_logfile
+
+input_directory = 'input'
+output_directory = 'output'
+
+trf_filepaths = glob(os.path.join(input_directory, '*.trf'))
+first_dicom_filepath = glob(os.path.join(input_directory, '*.dcm'))[0]
```

Comment:
This is an interesting way to do it. The user provides a DICOM as a template, basically?

Is the user given instructions to that effect? Haven't found them so far, but I might have just missed them!

Reply:
The user script isn't live yet :). I have commented out the option. Left it for another pull request.

Reply:
But yes, that's right. I've explained things on the app page:

![image](https://user-images.githubusercontent.com/6559099/58260169-52051f80-7db9-11e9-952a-46e117989adf.png)